samples/csharp_dotnetcore/84.bot-authentication-certificate/README.md

```diff
@@ -164,6 +164,11 @@ An SSL/TLS certificate is a digital object that allows systems to verify identit
 
       ![Certificate Reading](Images/KeyVault/CertificateReading.png)
```

Comment:
Case sensitivity of links is important if the repo is cloned in linux/unix systems. `Images` is not the same as `images`

Reply:
in this case the capital letter matches, in JS it doesn't